flink/v1.13/flink/src/main/java/org/apache/iceberg/flink/FlinkCatalogFactory.java

```diff
@@ -153,6 +156,17 @@ private static Configuration mergeHiveConf(Configuration hadoopConf, String hive
         newConf.addResource(configFile);
       }
     }
+
+    if (!Strings.isNullOrEmpty(hadoopConfDir)) {
+      if (new File(hadoopConfDir).exists()) {
```

Comment:
This should not ignore the option if it doesn't exist.

Reply:
Do you mean that if this configuration is added, and the directory is not exist,  the error should be reported normally? maybe I can remove this line of code

Reply:
Yeah, if there is a problem, this should raise an exception rather than ignoring the config dir.